test/unit/org/apache/cassandra/net/ConnectionUtils.java

```diff
@@ -96,6 +93,11 @@ public OutboundCountChecker error(long count, long bytes)
             return this;
         }
 
+        public void longCheck(long timeout, TimeUnit timeUnit)
```

Comment:
why not just `check`?  you don't have a signature conflict so could have the same API but with the duration.

Reply:
Picking the name mainly for clarity that the check is a long operation that can run for the duration specified.